### PR TITLE
Dropdown lists for animal color, age, size, gender, goodWith, coatLen…

### PR DESCRIPTION
**Pull request by AlonaVladymyrovaTrinity**
…gth, careAndBehavior  implemented

**The changes made in this commit include the implementation of dropdown lists for animal color, age, size, gender, goodWith, coatLength, and careAndBehavior. Below is a summary of the changes:**

**In the petAction.js file:**
- Added two new functions: getCatColors and getDogColors. These functions make API calls to retrieve cat colors and dog colors, respectively.
- Dispatch success or failure actions based on the API response.

**In the Pets.js component:**
- Imported the reducers catColorsReducer and dogColorsReducer.
- Added useEffect hooks for fetching cat colors and dog colors based on the selected type.
- Created options objects for breed, color, age, size, gender, goodWith, coatLength, and careAndBehavior dropdowns.
- Added event handlers for selecting breed, color, age, size, gender, goodWith, coatLength, and careAndBehavior.
- Updated the render function to include the new dropdowns and display the selected values.

In the petReducer.js file:
- Added two functions dogColorsReducer and catColorsReducer

**Testing Steps:**

- To test the changes made in this commit, follow these steps:
- Navigate to the Pets component in the application. [http://localhost:3000/pets](http://localhost:3000/pets)
- Verify that the dropdown lists for animal color, age, size, gender, goodWith, coatLength, and careAndBehavior are displayed correctly.
- Test the functionality of each dropdown:
- Select different options for each dropdown and ensure that the selected value is updated correctly.
- Verify that the options for breed, color, age, size, gender, goodWith, coatLength, and careAndBehavior are populated based on the selected animal type (Cat or Dog).

**Test the API calls:**

- Check the network requests in the browser's developer tools to confirm that the requests for cat colors and dog colors are being made correctly.
- Verify that the API responses for cat colors and dog colors are handled correctly by checking the corresponding reducers and state variables.
- Test any related functionality or components that depend on the changes made.
  
Screenshot 2023-06-17 at 9 41 12 PM
Screenshot 2023-06-17 at 9 41 50 PM

### CODE REVIEW

**Review by LiubovCass — APPROVED** [review]
Looks Awesome Alona!

File: src/actions/petAction.js [hunk 1]
```diff
@@ -83,3 +83,49 @@ export const getDogBreeds = async (dispatch) => {
     });
   }
 };
+
+// Get Cat Colors
+export const getCatColors = async (dispatch) => {
+  try {
+    dispatch({ type: 'CAT_COLORS_REQUEST' });
+    const response = await axios.get(`/api/v1/cat-colors`, {
+      withCredentials: true,
```
Comment (OxiBo) [inline src/actions/petAction.js]: You do not have to send `withCredentials: true` for unprotected api endpoints.

**Review by OxiBo — APPROVED** [review]
You ave done a lot of work. Everything looks great and works as expected for this point!

File: src/actions/petAction.js (hunk 1 above)
Comment (AlonaVladymyrovaTrinity) [reply src/actions/petAction.js]: withCredentials: true was removed for unprotected api endpoints.

**Pull request merged**